resource-managers/yarn/src/main/scala/org/apache/spark/deploy/yarn/Client.scala

```diff
@@ -86,6 +86,13 @@ private[spark] class Client(
     sparkConf.get(AM_CORES)
   }
 
+  private val amResources: collection.immutable.Map[String, String] =
```

Comment:
I think you can just write `Map`. Or omit the type too.

Reply:
fixed